### PR TITLE
namepace update from CentralMail to BenefitsIntake

### PR DESCRIPTION
**Pull request by mreed101**
CentralMail namespaces within the 686 submission workflow have been updated to BenefitsIntake.

## Summary

- *This work is behind a feature toggle (flipper): NO*
- *CentralMail for for 686 has been renamed to BenefitsIntake to clear up confusion as we use lighthouse for handling submissions*
- *BenefitsIntake is more consistent and allows us to change service later on without the need to rename services or modules again*
- *VA Benefits and Claims*
- *(If introducing a flipper, what is the success criteria being targeted?)*

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/92235

## Testing done

- [X] *New code is covered by unit tests*
- *The apps behavior does not change*
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
 - *This is not a new feature*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*Impacts are possible to form 686c submissions*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

We should throughly test the 686c workflow to ensure that all forms are being submitted properly and that logging is still accurate.

### CODE REVIEW

**Review by evansmith — CHANGES_REQUESTED** [review]
@mreed101 can you go through app/services/bgs/dependent_service.rb, app/sidekiq/bgs/submit_form674_job.rb, and app/sidekiq/bgs/submit_form686c_job.rb to clear out the references to `CentralMail::SubmitCentralForm686cJob`

Also there are a few logs that link to datadog dashboards that reference `CentralMail::SubmitCentralForm686cJob`. Should those messages and dashboards be updated?

**Review by evansmith — DISMISSED** [review]
LGTM, thanks for adding those changes!

**Review by rjohnson2011 — APPROVED** [review]

**Review by rjohnson2011 — APPROVED** [review]

File: spec/lib/dependents/monitor_spec.rb [hunk 1]
```diff
@@ -66,7 +66,7 @@
     it 'logs sidekiq job exhaustion with failure avoided' do
       msg = { 'args' => [claim.id, encrypted_vet_info, encrypted_user], error_message: 'Error!' }
 
-      log = "Failed all retries on CentralMail::SubmitCentralForm686cJob, last error: #{msg['error_message']}"
+      log = "Failed all retries on Lighthouse::BenefitsIntake::SubmitCentralForm686cJob, last error: #{msg['error_message']}" # rubocop:disable Layout/LineLength
```
Comment (ericboehs) [inline spec/lib/dependents/monitor_spec.rb:69]: Could you expand these please?
```suggestion
      log = 'Failed all retries on Lighthouse::BenefitsIntake::SubmitCentralForm686cJob, ' \
            "last error: #{msg['error_message']}"
```

Comment (mreed101) [reply spec/lib/dependents/monitor_spec.rb]: This was modified @ericboehs, please take another look if you get a chance.

File: lib/dependents/monitor.rb [hunk 2]
```diff
@@ -38,7 +38,7 @@ def track_submission_exhaustion(msg, email = nil)
 
       StatsD.increment("#{SUBMISSION_STATS_KEY}.exhausted")
       Rails.logger.error(
-        "Failed all retries on CentralMail::SubmitCentralForm686cJob, last error: #{msg['error_message']}"
+        "Failed all retries on Lighthouse::BenefitsIntake::SubmitCentralForm686cJob, last error: #{msg['error_message']}" # rubocop:disable Layout/LineLength
```
Comment (ericboehs) [inline lib/dependents/monitor.rb:41]: Here's another line that could be split up to avoid skirting the rubocop check

Comment (mreed101) [reply lib/dependents/monitor.rb]: This has been resolved in my latest push @ericboehs

**Review by ericboehs — APPROVED** [review]

**Review by ryan-mcneil — APPROVED** [review]

**Pull request merged**